sqlobject/src/main/java/org/jdbi/v3/sqlobject/locator/SqlLocator.java

```diff
@@ -29,6 +29,21 @@ public interface SqlLocator {
      * @param method the method
      * @param config the config registry
      * @return the SQL for the given method.
+     * @deprecated use {@link #locate(Class, Object, Method, ConfigRegistry)}
      */
+    @Deprecated
```

Comment:
This interface method should not be deprecated. None of the current implementations uses the new method and removing it would be a breaking change.

The retrofit is a bit annoying and it makes implementations of the new method with the target object require to also override the old method (e.g. with a "throws UnsupportedException").